.changeset/spicy-donuts-search.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/core': patch
+---
+
+Made Workspace.rebuildSearchIndex public so the search index can be built without init() starting the sandbox. Paths default to the configured autoIndexPaths.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)